autoPyTorch/pipeline/components/preprocessing/image_preprocessing/normalise/__init__.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
The series of changes you made should be achieved by writing the following in each `__init__.py`:
```
from autoPyTorch.pipeline.components.<something 1>.<something 2>.<something 3> import base_<something 4>_choice

<something 5>Choice = base_<something 4>_choice.<something 5>Choice
```

If there are any intentions behind the changes, let me know the reasons as a response here.

Reply:
I wrote this in the PR description, but let me state it here again. The problem is that forkserver, when communicating the pre-loaded modules from the parent process to the forked server client (the entity that spawns new process) seems to require the `__init__.py`has the choices defined on it. I made this change to mimic how the pipeline is implemented in auto-sklearn.

Defining the choice in the init is not sufficient, as the problem is how python organized the loading of the modules. Yes, maybe we could do some intelligent sourcing the choices via relative paths, but the criteria that I follow was to be similar to auto-sklearn so that we share both the same code (and solution to problems).

Reply:
> I made this change to mimic how the pipeline is implemented in auto-sklearn.

It is not a good practice to write many codes in `__init__.py`.
We should mimic good practices, but not bad practices. 

> Defining the choice in the init is not sufficient

Could you give me the error you got when you run?
I do not get any errors when I modify the codes accordingly.